iosApp/iosApp/Pages/Promo/PromoScreenView.swift

```diff
@@ -50,10 +50,27 @@ struct PromoScreenView: View {
 
     @ViewBuilder
     var combinedStopAndTrip: some View {
+        let promoDetailsKey = String(format: NSLocalizedString(
```

Comment:
I think `String(format:)` is only necessary for strings with templated values, but this is a fully static string and so `NSLocalizedString()` will return the full string already.